checkov/cloudformation/graph_builder/variable_rendering/renderer.py

```diff
@@ -0,0 +1,206 @@
+                    if evaluated_value and evaluated_value != original_value:
+                        val_to_eval[cfn_evaluation_function] = evaluated_value
+                        self.update_evaluated_value(
+                            changed_attribute_key=edge.label,
+                            changed_attribute_value=evaluated_value,
+                            vertex=edge.origin,
+                            change_origin_id=edge.dest,
+                            attribute_at_dest=edge.label,
+                        )
+
+    @staticmethod
+    def _evaluate_ref_connection(value: str, dest_vertex_attributes) -> Optional[str]:
+        evaluated_value = None
+
+        # in case of Ref we take only Parameter's default value
+        if value == dest_vertex_attributes.get(CustomAttributes.BLOCK_NAME) and dest_vertex_attributes.get(
+                CustomAttributes.BLOCK_TYPE) == BlockType.PARAMETERS:
+            evaluated_value = dest_vertex_attributes.get('Default', None)
+            if evaluated_value:
+                evaluated_value = str(evaluated_value)
+
+        return evaluated_value
+
+    @staticmethod
+    def _evaluate_findinmap_connection(value: List[str], dest_vertex_attributes) -> Optional[str]:
+        evaluated_value = None
+
+        # value = [ "MapName", "TopLevelKey", "SecondLevelKey"]
+        if isinstance(value, list) and len(value) == 3:
+            map_name = value[0]
+            top_level_key = value[1]
+            second_level_key = value[2]
+
+            if all(isinstance(element, str) for element in value) and \
+                    map_name == dest_vertex_attributes.get(CustomAttributes.BLOCK_NAME) and \
+                    dest_vertex_attributes.get(CustomAttributes.BLOCK_TYPE) == BlockType.MAPPINGS:
+                evaluated_value = dest_vertex_attributes.get(f'{top_level_key}.{second_level_key}', None)
+                if evaluated_value:
+                    evaluated_value = str(evaluated_value)
+
+        return evaluated_value
+
+    @staticmethod
+    def _evaluate_getatt_connection(value: List[str], dest_vertex_attributes) -> Optional[str]:
+        evaluated_value = None
+
+        # value = [ "logicalNameOfResource", "attributeName" ]
+        if isinstance(value, list) and len(value) == 2:
+            resource_name = value[0]
+            attribute_name = value[1]
+            dest_name = dest_vertex_attributes.get(CustomAttributes.BLOCK_NAME).split('.')[-1]
+
+            if all(isinstance(element, str) for element in value) and \
+                    resource_name == dest_name and \
+                    dest_vertex_attributes.get(CustomAttributes.BLOCK_TYPE) == BlockType.RESOURCE:
+                evaluated_value = dest_vertex_attributes.get(attribute_name, None)  # we extract only build time atts, not runtime
+                if evaluated_value:
+                    evaluated_value = str(evaluated_value)
+
+        return evaluated_value
```

Comment:
this could also be rewritten similar to my suggestion in `_evaluate_ref_connection()`